distribution/lib/Standard/Base/0.0.0-dev/src/Runtime.enso

```diff
@@ -161,6 +164,7 @@ type Context
         case self of
             Input  -> "Input"
             Output -> "Output"
+            Dataflow_Stack_Trace -> "Dataflow_Stack_Trace "
```

Comment:
Why is there a space at the end of the text literal?

Reply:
Removed.